acquia_lift.admin.inc

```diff
@@ -936,7 +895,7 @@ function acquia_lift_target_audience_save($label, $agent_name, $contexts, $strat
     $option_set->targeting[$machine_name] = array();
   }
   $option_set->targeting[$machine_name]['label'] = $label;
-  $option_set->targeting[$machine_name]['weight'] = count($option_set->targeting);
+  $option_set->targeting[$machine_name]['weight'] = $weight;
```

Comment:
Plus once we add in the dragons, it will be calculated automatically anyway rather than having users enter a weight.